Tasks/VsTestV2/distributedtest.ts

```diff
@@ -99,9 +99,16 @@ export class DistributedTest {
             if (this.inputDataContract.TestSelectionSettings.TestSelectionType.toLowerCase() !== 'testassemblies') {
                 sourceFilter = ['**\\*', '!**\\obj\\*'];
             }
-
+            const taskProps: { [key: string]: any; } = {MiniMatchLines: sourceFilter.length};
+            ci.publishTelemetry('TestExecution','MinimatchFilterPerformance', taskProps);
```

Comment:
don't publish it here, accumulate these properties in the dictionary and call ci.publishTelemetry once you have all the data